source/mainnet/smart-contracts/tutorials/sft-minting/build-smart-contract.rst

```diff
@@ -0,0 +1,162 @@
+            // Sender Account Args
+            .requiredOption("--sign-key <signKey>", "Account Signing Key")
+            .requiredOption("--sender <sender>", "Sender Account Address. This should be the owner of the Contract")
+            // Node Client args
+            .requiredOption("--auth-token <authToken>", "Concordium Node Auth Token", "rpcadmin")
+            .requiredOption("--ip <ip>", "Concordium Node IP", "127.0.0.1")
+            .requiredOption("--port <port>", "Concordium Node Port", (v) => parseInt(v), 10001)
+            .requiredOption("--timeout <timeout>", "Concordium Node request timeout", (v) => parseInt(v), 15000)
+            .action(
+                async (args: DeployModuleArgs) =>
+                await sendAccountTransaction(
+                    args,
+                    args.sender,
+                    args.signKey,
+                    // payload
+                    { content: Buffer.from(readFileSync(args.wasm)) } as DeployModulePayload,
+                    // Transaction Type
+                    AccountTransactionType.DeployModule,
+                ),
+            )
+        );
+    }
+    setupCliDeployModule(cli);
+
+Run the command below in order to deploy the contract that was built in the previous step.
+
+.. code-block:: console
+
+    ts-node ./src/cli.ts deploy --wasm ../dist/smart-contract-multi/module.wasm.v1 --sender $ACCOUNT --sign-key $SIGN_KEY
+
+If you have the output below, you’ve successfully deployed your semi-fungible token smart contract on Concordium.
+
+.. image:: images/contract-deploy-success.png
+    :width: 100%
+
+You can also verify it either by looking at `CCDScan <https://ccdscan.io>`_ or the `testnet dashboard lookup section <https://dashboard.testnet.concordium.com/>`_. Now you need to go to the dashboard and get the hash value from there using the URL in the terminal. Simply copy the link from the terminal and visit it to look at the status of your transaction. Click **Deployed module with reference** and copy the hash value. You will use it while initializing the contract in the next section.
+
+.. image:: images/contract-deploy-success-db.png
+    :width: 100%
+
+And you can see it in the wallet.
+
+.. image:: images/contract-deploy-success-wallet.png
+    :width: 100%
+
+Initialize the contract
+=======================
+
+Now you need to initialize the deployed contract. It’s a lot easier than the previous steps. After deploying a contract you have to initialize it; it’s like object-oriented programming. You create a class which is a module, and then you initialize it to create an object. An object of a class is a way to store both states of the class and its functionality. This time you are going to use the hash value you got in the previous step. First, make sure initialize function is implemented in your cli.ts file as shown below.
+
+.. code-block:: console
```

Comment:
Is this section a leftover? Below it says that we will be using concordium-client.

Reply:
Yes, because I had not finished editing it yet.